cernopendata_client/cli.py

```diff
@@ -161,6 +185,9 @@ def download_files(server, recid, doi, title, protocol, expand):
     \b
     Examples:
       $ cernopendata-client download-files --recid 5500
+      $ cernopendata-client download-files --recid 1 --filter-name 00E081791775.root
+      $ cernopendata-client download-files --recid 1 --filter-regexp E011
```

Comment:
Here too for filter regexp you can use 5500 and '\.py$' perhaps? (to download all Python files could be a good example)